packages/upgrade-tests/tsconfig.build.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
Do we need this config? Will this ever be "built for production"?

Reply:
Good question, I was copying the layout of our other packages. `tsconfig.json` depends on this file currently so I think strictly yes? Though if you think this should be structured differently let me know how and I'll do it.